### PR TITLE
change env example to be a program and do a username,password thing

### PR DESCRIPTION
**Pull request by steverusso**

### CODE REVIEW

File: examples/simple_env/main.go [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+// Any changes to this file likely necessitate changes to the project's README.
+//go:generate goclap -type mycli
+
+import (
+	"fmt"
+	"os"
+)
+
+// Print a string with the option to make it uppercase.
+type mycli struct {
```
Comment (russoj88) [inline examples/simple_env/main.go:12]: I don't think this example should include command line options. It takes away from the purpose to show an environment variable example.

Comment (steverusso) [reply examples/simple_env/main.go:12]: Hmm, if not options though, then they'd default to being optional positional arguments. I think that's a less thorough example, but changing one of them might be the right balance.

File: examples/simple_env/main.go [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+	// clap:opt prefix,p
+	// clap:env MY_PREFIX
+	prefix string
+	// The user provided input.
+	//
+	// clap:env MY_INPUT
+	input string
+}
+
+func main() {
```
Comment (russoj88) [inline examples/simple_env/main.go:24]: How does this example show using env variables?

**Review by russoj88 — APPROVED** [review]

File: examples/simple_env/main.go (hunk 2 above)
Comment (steverusso) [reply examples/simple_env/main.go:24]: Good point, actually none of the examples technically show usage. Tracking with https://github.com/steverusso/goclap/issues/53.

**Pull request merged**